2024.html

```diff
@@ -86,6 +86,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://felixwaller.dev/">Felix Waller</a></li>
             <li><a href="https://blakewatson.com/">Blake Watson</a></li>
             <li><a href="https://cssence.com/">Matthias Zöchling</a></li>
+            <li><a href="https://kayserifserif.place/">Katherine Yang</a></li>
```

Comment:
Please move one up, before Matthias (to keep it in alphabetical order) :)

Reply:
oh thank you! sorry i didn't catch that. fixed!